ycmd/completers/cs/cs_completer.py

```diff
@@ -702,6 +704,8 @@ def _GetDoc( self, request_data ):
     if ( result[ "Documentation" ] ):
       message += "\n" + result[ "Documentation" ]
 
+    if not message:
```

Comment:
@bstaletic I think you should do this before trying to concatenate with `result[ "Documentation" ]` (in the lines above), as you can't assume both should always come out as `None` in this situation - that's an implementation detail if it's the case, not a invariant.

Reply:
I did consider that. The reason I didn't do it the way you suggest is because, logically, how can there be a documentation for something that has an unknown type? I was also unable to get Roslyn to return non-null `Documentation`, but null `Type`.

To be honest, I mostly didn't want to mess with mocking a response that I can't actually get from the server.

Reply:
Actually, I've made some changes. Instead of doing `message = result[ 'Type' ]`, now we're doing `message = result.get( 'Type' ) or ''`. This way the concatenation will always be safe and in the off chance that there's no type, but there is documentation, `.strip()` will strip the leading new line.

Reply:
That's nice! Thank you